<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>872</width>
    <height>324</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Taschenrechner</string>
  </property>
  <property name="styleSheet">
   <string notr="true">background-color:rgb(95, 97, 99)</string>
  </property>
  <property name="animated">
   <bool>true</bool>
  </property>
  <widget class="QWidget" name="centralwidget">
   <property name="minimumSize">
    <size>
     <width>872</width>
     <height>0</height>
    </size>
   </property>
   <property name="autoFillBackground">
    <bool>false</bool>
   </property>
   <property name="styleSheet">
    <string notr="true"/>
   </property>
   <layout class="QGridLayout" name="gridLayout">
    <item row="4" column="7">
     <widget class="QPushButton" name="zahl5">
      <property name="styleSheet">
       <string notr="true">background-color: rgb(122, 123, 123);
color: rgb(255, 255, 255);</string>
      </property>
      <property name="text">
       <string>5</string>
      </property>
     </widget>
    </item>
    <item row="5" column="1">
     <widget class="QPushButton" name="cosinus">
      <property name="styleSheet">
       <string notr="true">color: rgb(255, 255, 255);</string>
      </property>
      <property name="text">
       <string>cos</string>
      </property>
     </widget>
    </item>
    <item row="2" column="7">
     <widget class="QPushButton" name="zahl8">
      <property name="styleSheet">
       <string notr="true">background-color: rgb(122, 123, 123);
color: rgb(255, 255, 255);</string>
      </property>
      <property name="text">
       <string>8</string>
      </property>
     </widget>
    </item>
    <item row="2" column="3">
     <widget class="QPushButton" name="wurzel">
      <property name="styleSheet">
       <string notr="true">color: rgb(255, 255, 255);</string>
      </property>
      <property name="text">
       <string>√x</string>
      </property>
     </widget>
    </item>
    <item row="5" column="9">
     <widget class="QPushButton" name="addition">
      <property name="styleSheet">
       <string notr="true">background-color: rgb(255, 159, 12);
color: rgb(255, 255, 255);</string>
      </property>
      <property name="text">
       <string>+</string>
      </property>
     </widget>
    </item>
    <item row="4" column="1">
     <widget class="QPushButton" name="sinus">
      <property name="styleSheet">
       <string notr="true">color: rgb(255, 255, 255);</string>
      </property>
      <property name="text">
       <string>sin</string>
      </property>
     </widget>
    </item>
    <item row="1" column="6">
     <widget class="QPushButton" name="reset">
      <property name="styleSheet">
       <string notr="true">color: rgb(255, 255, 255);</string>
      </property>
      <property name="text">
       <string>AC</string>
      </property>
     </widget>
    </item>
    <item row="4" column="8">
     <widget class="QPushButton" name="zahl6">
      <property name="styleSheet">
       <string notr="true">background-color: rgb(122, 123, 123);
color: rgb(255, 255, 255);</string>
      </property>
      <property name="text">
       <string>6</string>
      </property>
     </widget>
    </item>
    <item row="4" column="9">
     <widget class="QPushButton" name="subtraktion">
      <property name="styleSheet">
       <string notr="true">background-color: rgb(255, 159, 12);
color: rgb(255, 255, 255);</string>
      </property>
      <property name="text">
       <string>-</string>
      </property>
     </widget>
    </item>
    <item row="1" column="9">
     <widget class="QPushButton" name="division">
      <property name="styleSheet">
       <string notr="true">background-color: rgb(255, 159, 12);
color: rgb(255, 255, 255);</string>
      </property>
      <property name="text">
       <string>/</string>
      </property>
     </widget>
    </item>
    <item row="4" column="5">
     <widget class="QPushButton" name="kubischx">
      <property name="styleSheet">
       <string notr="true">color: rgb(255, 255, 255);</string>
      </property>
      <property name="inputMethodHints">
       <set>Qt::ImhNone</set>
      </property>
      <property name="text">
       <string>x^3</string>
      </property>
     </widget>
    </item>
    <item row="6" column="2">
     <widget class="QPushButton" name="pi">
      <property name="styleSheet">
       <string notr="true">color: rgb(255, 255, 255);</string>
      </property>
      <property name="text">
       <string>π</string>
      </property>
     </widget>
    </item>
    <item row="2" column="9">
     <widget class="QPushButton" name="multiplikation">
      <property name="styleSheet">
       <string notr="true">background-color: rgb(255, 159, 12);
color: rgb(255, 255, 255);</string>
      </property>
      <property name="text">
       <string>x</string>
      </property>
     </widget>
    </item>
    <item row="6" column="6" colspan="2">
     <widget class="QPushButton" name="zahl0">
      <property name="styleSheet">
       <string notr="true">background-color: rgb(122, 123, 123);
color: rgb(255, 255, 255);</string>
      </property>
      <property name="text">
       <string>0</string>
      </property>
     </widget>
    </item>
    <item row="5" column="5">
     <widget class="QPushButton" name="quadratx">
      <property name="styleSheet">
       <string notr="true">color: rgb(255, 255, 255);</string>
      </property>
      <property name="text">
       <string>x^2</string>
      </property>
     </widget>
    </item>
    <item row="6" column="3">
     <widget class="QPushButton" name="ntewurzel">
      <property name="styleSheet">
       <string notr="true">color: rgb(255, 255, 255);</string>
      </property>
      <property name="text">
       <string>ysqrt(n)</string>
      </property>
     </widget>
    </item>
    <item row="5" column="8">
     <widget class="QPushButton" name="zahl3">
      <property name="styleSheet">
       <string notr="true">background-color: rgb(122, 123, 123);
color: rgb(255, 255, 255);</string>
      </property>
      <property name="text">
       <string>3</string>
      </property>
     </widget>
    </item>
    <item row="2" column="6">
     <widget class="QPushButton" name="zahl7">
      <property name="styleSheet">
       <string notr="true">background-color: rgb(122, 123, 123);
color: rgb(255, 255, 255);</string>
      </property>
      <property name="text">
       <string>7</string>
      </property>
     </widget>
    </item>
    <item row="1" column="5">
     <widget class="QPushButton" name="fib">
      <property name="styleSheet">
       <string notr="true">color: rgb(255, 255, 255);</string>
      </property>
      <property name="text">
       <string>fib</string>
      </property>
     </widget>
    </item>
    <item row="6" column="1">
     <widget class="QPushButton" name="tangens">
      <property name="styleSheet">
       <string notr="true">color: rgb(255, 255, 255);</string>
      </property>
      <property name="text">
       <string>tan</string>
      </property>
     </widget>
    </item>
    <item row="4" column="2">
     <widget class="QPushButton" name="eulerhochx">
      <property name="styleSheet">
       <string notr="true">color: rgb(255, 255, 255);</string>
      </property>
      <property name="text">
       <string>e^x</string>
      </property>
     </widget>
    </item>
    <item row="5" column="2">
     <widget class="QPushButton" name="euler">
      <property name="styleSheet">
       <string notr="true">color: rgb(255, 255, 255);</string>
      </property>
      <property name="text">
       <string>e</string>
      </property>
     </widget>
    </item>
    <item row="1" column="7">
     <widget class="QPushButton" name="negation">
      <property name="styleSheet">
       <string notr="true">color: rgb(255, 255, 255);</string>
      </property>
      <property name="text">
       <string>+/-</string>
      </property>
     </widget>
    </item>
    <item row="4" column="6">
     <widget class="QPushButton" name="zahl4">
      <property name="styleSheet">
       <string notr="true">background-color: rgb(122, 123, 123);
color: rgb(255, 255, 255);</string>
      </property>
      <property name="text">
       <string>4</string>
      </property>
     </widget>
    </item>
    <item row="2" column="8">
     <widget class="QPushButton" name="zahl9">
      <property name="styleSheet">
       <string notr="true">background-color: rgb(122, 123, 123);
color: rgb(255, 255, 255);</string>
      </property>
      <property name="text">
       <string>9</string>
      </property>
     </widget>
    </item>
    <item row="1" column="8">
     <widget class="QPushButton" name="prozent">
      <property name="styleSheet">
       <string notr="true">color: rgb(255, 255, 255);</string>
      </property>
      <property name="text">
       <string>%</string>
      </property>
     </widget>
    </item>
    <item row="6" column="8">
     <widget class="QPushButton" name="komma">
      <property name="styleSheet">
       <string notr="true">background-color: rgb(122, 123, 123);
color: rgb(255, 255, 255);</string>
      </property>
      <property name="text">
       <string>,</string>
      </property>
     </widget>
    </item>
    <item row="6" column="9">
     <widget class="QPushButton" name="gleich">
      <property name="styleSheet">
       <string notr="true">background-color: rgb(255, 159, 12);
color: rgb(255, 255, 255);</string>
      </property>
      <property name="text">
       <string>=</string>
      </property>
     </widget>
    </item>
    <item row="5" column="7">
     <widget class="QPushButton" name="zahl2">
      <property name="styleSheet">
       <string notr="true">background-color: rgb(122, 123, 123);
color: rgb(255, 255, 255);</string>
      </property>
      <property name="text">
       <string>2</string>
      </property>
      <property name="default">
       <bool>false</bool>
      </property>
     </widget>
    </item>
    <item row="2" column="5">
     <widget class="QPushButton" name="xhochy">
      <property name="styleSheet">
       <string notr="true">color: rgb(255, 255, 255);</string>
      </property>
      <property name="text">
       <string>x^y</string>
      </property>
     </widget>
    </item>
    <item row="5" column="6">
     <widget class="QPushButton" name="zahl1">
      <property name="styleSheet">
       <string notr="true">background-color: rgb(122, 123, 123);
color: rgb(255, 255, 255);</string>
      </property>
      <property name="text">
       <string>1</string>
      </property>
     </widget>
    </item>
    <item row="2" column="2">
     <widget class="QPushButton" name="betrag">
      <property name="styleSheet">
       <string notr="true">color: rgb(255, 255, 255);</string>
      </property>
      <property name="text">
       <string>|x|</string>
      </property>
     </widget>
    </item>
    <item row="5" column="3">
     <widget class="QPushButton" name="viertewurzel">
      <property name="styleSheet">
       <string notr="true">color: rgb(255, 255, 255);</string>
      </property>
      <property name="text">
       <string>∜x</string>
      </property>
     </widget>
    </item>
    <item row="4" column="3">
     <widget class="QPushButton" name="drittewurzel">
      <property name="styleSheet">
       <string notr="true">color: rgb(255, 255, 255);</string>
      </property>
      <property name="text">
       <string>∛x</string>
      </property>
     </widget>
    </item>
    <item row="2" column="1">
     <widget class="QPushButton" name="exp">
      <property name="styleSheet">
       <string notr="true">color: rgb(255, 255, 255);</string>
      </property>
      <property name="text">
       <string>exp(x)</string>
      </property>
     </widget>
    </item>
    <item row="6" column="5">
     <widget class="QPushButton" name="standardhyperbel">
      <property name="styleSheet">
       <string notr="true">color: rgb(255, 255, 255);</string>
      </property>
      <property name="text">
       <string>1/x</string>
      </property>
     </widget>
    </item>
    <item row="6" column="4">
     <widget class="QPushButton" name="logxy">
      <property name="styleSheet">
       <string notr="true">color: rgb(255, 255, 255);</string>
      </property>
      <property name="text">
       <string>logxy</string>
      </property>
     </widget>
    </item>
    <item row="5" column="4">
     <widget class="QPushButton" name="log10">
      <property name="styleSheet">
       <string notr="true">color: rgb(255, 255, 255);</string>
      </property>
      <property name="text">
       <string>log10</string>
      </property>
     </widget>
    </item>
    <item row="4" column="4">
     <widget class="QPushButton" name="natlogarithmus">
      <property name="styleSheet">
       <string notr="true">color: rgb(255, 255, 255);</string>
      </property>
      <property name="text">
       <string>ln</string>
      </property>
     </widget>
    </item>
    <item row="2" column="4">
     <widget class="QPushButton" name="fak">
      <property name="styleSheet">
       <string notr="true">color: rgb(255, 255, 255);</string>
      </property>
      <property name="text">
       <string>n!</string>
      </property>
     </widget>
    </item>
    <item row="1" column="4">
     <widget class="QPushButton" name="ggt">
      <property name="styleSheet">
       <string notr="true">color: rgb(255, 255, 255);</string>
      </property>
      <property name="text">
       <string>ggT</string>
      </property>
     </widget>
    </item>
    <item row="1" column="3">
     <widget class="QPushButton" name="kgv">
      <property name="styleSheet">
       <string notr="true">color: rgb(255, 255, 255);</string>
      </property>
      <property name="text">
       <string>KgV</string>
      </property>
     </widget>
    </item>
    <item row="1" column="2">
     <widget class="QPushButton" name="klammerzu">
      <property name="styleSheet">
       <string notr="true">color: rgb(255, 255, 255);</string>
      </property>
      <property name="text">
       <string>)</string>
      </property>
     </widget>
    </item>
    <item row="1" column="1">
     <widget class="QPushButton" name="klammerauf">
      <property name="styleSheet">
       <string notr="true">color: rgb(255, 255, 255);</string>
      </property>
      <property name="text">
       <string>(</string>
      </property>
     </widget>
    </item>
    <item row="0" column="1" colspan="9">
     <widget class="QLCDNumber" name="Display">
      <property name="styleSheet">
       <string notr="true">background-color: rgb(77, 79, 80);
color: rgb(255, 255, 255);</string>
      </property>
     </widget>
    </item>
   </layout>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
